src/main/java/com/fdmgroup/forex/enums/OrderStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.fdmgroup.forex.enums;
+
+public enum OrderStatus {
+    ACTIVE, CANCELLED, CLEARED, CLOSED, EXPIRED
```

Comment:
Do you think it's a good idea to add JavaDoc here to explain what each enum mean?
Especially CANCELLED vs CLEARED vs CLOSED

Reply:
Yes, will do

Reply:
See [593c591](https://github.com/forex-agile/backend/pull/8/commits/593c59162fe1595128ade621afe5f4076d010789)